tests/file_storage/tests.py

```diff
@@ -908,20 +908,15 @@ def test_first_character_dot(self):
 
 class ContentFileStorageTestCase(unittest.TestCase):
 
-    def setUp(self):
```

Comment:
You refactored this test, but not other similar ones in this file. Probably these tests could be better organized and perhaps consolidated so there aren't so many test classes. Maybe leave this alone for now?